lib/node_modules/@stdlib/stats/base/svariancewd/benchmark/benchmark.ndarray.js

```diff
@@ -21,14 +21,20 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
-var Float32Array = require( '@stdlib/array/float32' );
+var uniform = require( '@stdlib/random/array/uniform' );
 var pkg = require( './../package.json' ).name;
 var svariancewd = require( './../lib/ndarray.js' );
 
 
+// VARIABLES //
+
+var option = {
+	'dtype': 'float32'
+};
```

Comment:
```suggestion
var options = {
	'dtype': 'float32'
};
```
just updating this as a matter of convention

Reply:
same comment for all other javascript benchmarks